suite-native/transactions/src/components/TransactionDetail/TransactionDetailHeader.tsx

```diff
@@ -41,21 +58,26 @@ export const TransactionDetailHeader = ({
 }: TransactionDetailHeaderProps) => {
     const { FiatAmountFormatter } = useFormatters();
 
-    if (type !== 'recv' && type !== 'sent')
-        return <ErrorMessage errorMessage={`Unknown transaction type ${type}.`} />;
+    const { text } = transactionTypeInfo[type];
+
+    const hasTransactionSign = type === 'sent' || type === 'recv';
```

Comment:
```suggestion
    const hasTransactionSign = 'sign' in transactionTypeInfo[type];
```

Reply:
Nice idea, altough that will turn of type guard for `sign` and `signColor` in text as it will create a `Partial<>` type. You can read more about this here https://www.typescriptlang.org/docs/handbook/advanced-types.html#mapped-types

Reply:
Ok, I wasn't aware of this